### PR TITLE
Updating supported platforms

### PR DESCRIPTION
**Pull request by KalyanJoseph**
Updated the information on the "Supported Platforms" page, to include that Temurin 8/11/16 are now available on x64 Linux, x64 Windows and x64 MacOS. Also changed "Adoptium 8/11/16" to "Temurin 8/11/16".

##### Checklist

- [ ] `npm test` passes
- [ ] documentation is changed or added (if applicable)
- [ ] permission has been obtained to add new logo (if applicable)
- [ ] contribution guidelines followed [here](https://github.com/adoptium/adoptium.net/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/handlebars/supported_platforms.handlebars [hunk 1]
```diff
@@ -8,15 +8,15 @@
 
   <div class="align-left support">
 
-    <p>We're currently working out our list of supported platforms. Please check back later!</p>
+    {{!-- <p>We're currently working out our list of supported platforms. Please check back later!</p> --}}
```
Comment (karianna) [inline src/handlebars/supported_platforms.handlebars]: I think you can remove this line entirely now

File: src/handlebars/supported_platforms.handlebars [hunk 2]
```diff
@@ -41,143 +41,145 @@
...
-                  <td><i class="fa fa-star-o hotspot" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
+                  <td><i class="fa fa-star-o hotspot" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td> --}}
                 </tr>
             </tbody>
         </table>
 
-        <p><strong>Note:</strong> Any other Linux distributions that have a minimum glibc version 2.12 on x86, 2.23 on aarch32 and 2.17 on other architectures are expected to function without problems.</p>
+        <br/>
 
+        {{!-- <p><strong>Note:</strong> Any other Linux distributions that have a minimum glibc version 2.12 on x86, 2.23 on aarch32 and 2.17 on other architectures are expected to function without problems.</p>
```
Comment (karianna) [inline src/handlebars/supported_platforms.handlebars]: I think you can uncomment that.

File: src/handlebars/supported_platforms.handlebars [hunk 3]
```diff
@@ -859,7 +863,197 @@
             <td><i class="fa fa-star-o hotspot" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
           </tr>
         </tbody>
-      </table>
+      </table> --}}
+
+
+  <h3 id="openjdk11_platforms">Temurin 16</h3>
```
Comment (karianna) [inline src/handlebars/supported_platforms.handlebars]: id should have 16 in it

Comment (karianna) [inline src/handlebars/supported_platforms.handlebars]: I think the whitespace is out

**Review by karianna — CHANGES_REQUESTED** [review]
Some nitpicks but looks good overall!

File: src/handlebars/supported_platforms.handlebars (hunk 1 above)
Comment (KalyanJoseph) [reply src/handlebars/supported_platforms.handlebars]: done!

**Review by karianna — APPROVED** [review]

**Pull request merged**